.github/workflows/terraform-fmt.yaml

```diff
@@ -0,0 +1,15 @@
+name: terraform-fmt
+run-name: terraform-fmt check ${{ github.event.pull_request.title }}
+on:
+  pull_request:
+    types: [ edited, assigned, opened, reopened, synchronize, ready_for_review ]
```

Comment:
Should we just use `opened`, `synchronize`, and `reopened` for these?

Reply:
I think `ready_for_review` makes sense (or as about as much as `reopened` does), but dropping `edited` and `assigned` should be OK.

Reply:
`opened` will run the changes.  This also happens if it is opened as a draft.

`ready_for_review` will trigger again, regardless of any changes, when the draft is set to ready.  Not sure if that gets us any value.

I feel like the three others will get most of the times we actually make a change.  May be worth testing though.